cli/src/merge_tools/builtin.rs

```diff
@@ -424,8 +425,133 @@ pub fn edit_diff_builtin(
     Ok(tree_id)
 }
 
+fn make_merge_sections(
+    merge_result: MergeResult,
+    file_merge: Merge<Option<FileId>>,
+) -> Result<Vec<scm_record::Section<'static>>, BuiltinToolError> {
+    let mut sections = Vec::new();
+    match merge_result {
+        MergeResult::Resolved(ContentHunk(buf)) => {
+            let right_id = file_merge.adds().get(1).and_then(|side| side.clone());
```

Comment:
Does this `1` suggest that this function only works with 2-sides conflicts? If so, add an assertion or return an error if we get a different input?

Reply:
I think this code was wrong as written before: it just assumed that the hash of the file, if the merge was resolved, would correspond to side `1` of the adds of the conflict. But the user could have selected side `0`, or somehow merged the results and gotten a hash that corresponds to neither side. The correct approach would be to hash the contents of `buf` here. But this is just used for display purposes, so, for now, I've just set the hash to `None`, so that it won't be rendered.